### DISCUSSION TITLE
[QUESTION] - Recommended resources to learn how to create a new vocabulary

### DISCUSSION THREAD

**Discussion by mlitkeok** [opencaesar/platform#12] [Q&A]
## Description
There are resources around that web that document how to create a new vocabulary, but I would be interested if there are any in particular you would recommend, thanks.

**Comment by melaasar** [comment]
If you are referring to how to create a new vocabulary in OML, then I am working now on this [issue](https://github.com/opencaesar/oml/issues/3) in this sprint, which will help with that. Please follow it to get updates.

If you are referring to how to create a new vocabulary in OWL, then the [OWL Functional Syntax](https://www.w3.org/TR/owl2-syntax/) is a good reference.
